internal/tasks/tasks.go

```diff
@@ -164,213 +164,208 @@ func ExecuteTask(task Task) Task {
 		log.Fatal(err.Error())
 	}
 
-	if diagnostics.GetReleaseVersion() == diagnostics.DEV_VERSION {
-		log.Printf("Dev environemnt. Not executing tasks.")
-	} else {
-		log.Println("Task: " + task.Task)
-		log.Println("Args: " + task.Args.String)
-		switch task.Task {
-		case "setup_backups":
-
-			log.Println("Setting up Backups Destination...")
-			var args taskSetupBackupsArgs
-			err := json.Unmarshal([]byte(task.Args.String), &args)
-			if err != nil {
-				log.Println("Error reading arguments of setup_backups task.")
-			} else {
-				taskResult := taskSetupBackups(args)
-				taskResultBool := true
-				// Check if returned taskResult string contains "error"
-				if strings.Contains(taskResult, "error") {
-					taskResultBool = false
-				}
-				task.Result = sql.NullString{String: taskResult, Valid: taskResultBool}
-			}
-
-		case "start_backup":
-
-			log.Println("Backing up Edgebox...")
-			taskResult := taskBackup()
-			taskResultBool := true
-			// Check if returned taskResult string contains "error"
-			if strings.Contains(taskResult, "error") {
-				taskResultBool = false
-			}
-			task.Result = sql.NullString{String: taskResult, Valid: taskResultBool}
-
-		case "restore_backup":
-			log.Println("Attempting to Restore Last Backup to Edgebox")
-			taskResult := taskRestoreBackup()
+	log.Println("Task: " + task.Task)
+	log.Println("Args: " + task.Args.String)
+	switch task.Task {
```

Comment:
This switch is getting huge :P Wondering if we could abstract this into something more manageable.